incubator/hnc/api/v1alpha2/hierarchy_types.go

```diff
@@ -31,11 +31,14 @@ const (
 
 // Constants for labels and annotations
 const (
-	MetaGroup                = "hnc.x-k8s.io"
-	LabelInheritedFrom       = MetaGroup + "/inheritedFrom"
-	FinalizerHasSubnamespace = MetaGroup + "/hasSubnamespace"
-	LabelTreeDepthSuffix     = ".tree." + MetaGroup + "/depth"
-	AnnotationManagedBy      = MetaGroup + "/managedBy"
+	MetaGroup                 = "hnc.x-k8s.io"
+	LabelInheritedFrom        = MetaGroup + "/inheritedFrom"
+	FinalizerHasSubnamespace  = MetaGroup + "/hasSubnamespace"
+	LabelTreeDepthSuffix      = ".tree." + MetaGroup + "/depth"
+	AnnotationManagedBy       = MetaGroup + "/managedBy"
```

Comment:
nit: maybe link issue to rename to `managed-by`?

Reply:
That's not really related to this change, but I'm working on it now anyway.